### ISSUE TITLE
Segmentation fault on X.Org since version 2024.5.0 (AppImage)

### ISSUE THREAD

**Issue by araujorm** [jitsi/jitsi-meet-electron#962]
## Description
Client (appimage) always crashes with segmentation fault since version 2024.5.0 (automatically downloaded yesterday, also tried downloading manually)

## Current behavior
Opens a blank window briefly, then crashes (segfault)
```
10:59:55.500 › Checking for update
Segmentation fault (core dumped)
```

## Expected Behavior
Not crashing and working as usual

## Possible Solution
Revert to 2024.3.0 or use any browser instead

## Steps to reproduce
Run latest 2024.5.0 jitsi-meet-x86_64.AppImage

# Environment details
Fedora Linux 39 (Thirty Nine)
Intel(R) Core(TM) i5-8265U CPU + 12 GB RAM
```
$ inxi -G
Graphics:
  Device-1: Intel WhiskeyLake-U GT2 [UHD Graphics 620] driver: i915 v: kernel
  Device-2: IMC Networks Integrated Camera driver: uvcvideo type: USB
  Display: x11 server: X.Org v: 1.20.14 with: Xwayland v: 23.2.6 driver: X:
    loaded: intel dri: iris gpu: i915 resolution: 1440x810~60Hz
  API: OpenGL v: 4.6 vendor: intel mesa v: 23.3.6 renderer: Mesa Intel UHD
    Graphics 620 (WHL GT2)
  API: Vulkan v: 1.3.275 drivers: N/A surfaces: xcb,xlib
```
No other crashes or system instability happening besides this

Also tried removing `$HOME/.config/Jitsi\ Meet/` but still crashes.

**Comment by saghul** [comment]
Can you get a backtrace?

**Comment by araujorm** [comment]
> Can you get a backtrace?

I tried with gdb but not very useful at the moment, I guess I'm missing some symbols? Any advice?
```
12:14:46.632 › Update for version 2024.5.0 is not available (latest version: 2024.5.0, downgrade is disallowed).
12:14:46.634 › checkForUpdatesAndNotify called, downloadPromise is null

Thread 1 "jitsi-meet" received signal SIGSEGV, Segmentation fault.
0x000055555cdd286e in ?? ()
(gdb) bt
#0  0x000055555cdd286e in ??? ()
#1  0x8000000080000000 in ??? ()
#2  0x0000333001010101 in ??? ()
#3  0x0000000000000000 in ??? ()
```

Meanwhile tried with other resolutions (since I know I usually don't use a very standard one...) but result is exactly the same at both `1600x900~60Hz` and `1920x1080~60Hz`.

Also tried (again) removing `$HOME/.config/Jitsi\ Meet/` and noticed that when I do it doesn't crash right away: I get the introductory message that I can skip, and it all seems well until I maximize the window. It always crashes then.

I'm using cinammon with a panel (taskbar alike) on top, tried with several sizes from 21 to 32 (usually I have 24) but no matter what, jitsi-meet-electron 2024.5.0 always crashes too when maximizing.

For now I'm reverting to 2024.3.0 and chown'ed `$HOME/.cache/jitsi-meet-electron-updater` as root to prevent auto-update, but if you can give me any advice on how to get a useful backtrace I can try to do so. Thanks in advance.

**Comment by saghul** [comment]
This is already very useful, thank you! Not sure how to get a more meaningful backtrrace TBH. The question is if this is a problem on Electron 30 or in something else...

**Comment by araujorm** [comment]
Meanwhile tried the Flathub version, doesn't crash. I'll be testing it as a workaround too.

**Comment by csett86** [comment]
And regarding maximizing, I found this very recent fix regarding crashes when maximizing on electron 30.0.4 (we are at 30.0.3), so maybe its then fixed by that as well: https://github.com/electron/electron/pull/42126

**Comment by saghul** [comment]
Good catch @csett86 !

**Comment by csett86** [comment]
And this fix in electron 30.0.5 looks promising as well: https://github.com/electron/electron/pull/42179
